Header header
float32 steeringNormalized
float32 accelerationNormalized